script/list-page.js

```diff
@@ -0,0 +1,205 @@
+const sidebar = document.querySelector("#sidebar-product");
+const listpage = document.querySelector("#list");
+const gridpage = document.querySelector("#grid");
+const show = document.querySelector("#show");
+const paginationID = document.querySelector("#pagination");
+let limit = 9;
+function start() {
```

Comment:
```suggestion
function init() {
```
thường thì ngta sẽ đặt tên như thế này, init khởi tạo ấy